### PR TITLE
Authentication Implementation

### PR DESCRIPTION
**Pull request by agnesnatasya**
Implement authentication and routing in the page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+<<<<<<< HEAD
```
Comment (yuheong) [inline package.json]: need to remove extra lines and @react-google-maps/api

**Comment by yuheong** [comment]
overall looks good except the erroneous lines in package.json!

**Comment by agnesnatasya** [comment]
ok updated, thanks Yuhe, but idk, i just realised, i cant really build it says react-leaflet is not found, i think it's some docker cache, can you try and see if it builds on you?

**Comment by yuheong** [comment]
hmm should be a docker thing - just try running npm install/yarn inside the docker container and it should resolve it.
Otherwise can just delete and restart the container!

**Review by yuheong — APPROVED** [review]

**Pull request merged**